syncmaster/schemas/v1/transfers/file_format.py

```diff
@@ -15,6 +17,47 @@
 )
 
 
+class ORCCompression(str, Enum):
+    UNCOMPRESSED = "uncompressed"
+    NONE = "none"
+    SNAPPY = "snappy"
+    ZLIB = "zlib"
+    LZ4 = "lz4"
+
+
+class ParquetCompression(str, Enum):
+    UNCOMPRESSED = "uncompressed"
+    NONE = "none"
```

Comment:
What's the difference?

Reply:
The options are the same: "none" is preferred, but "uncompressed" is used for compatibility. Got options from docs

Reply:
I mean, this is REST API schema, not some sort of Spark Enum. These values should make sense for API users, and not bound to some legacy Spark stuff. We can convert these values to Spark-compatible ones on worker side.